docs/designers-developers/developers/tutorials/create-block/wp-plugin.md

```diff
@@ -108,6 +108,7 @@ function create_block_gutenpride_block_init() {
 		$script_asset['dependencies'],
 		$script_asset['version']
 	);
+	wp_set_script_translations( 'create-block-gutenpride-block-editor', 'create_block' );
```

Comment:
Based on the text domain in the file header this should be `create-block`. But: The [current default value](https://github.com/WordPress/gutenberg/blob/2a8872c49bab0bf6cbfedb29b563bcdcdabb58fe/packages/create-block/lib/scaffold.js#L57) is wrong IMO, it should match the (unique) slug of the plugin and not use the namespace.

Reply:
Good catch, I was basing it off another example, which I updated in 9796019441441539e2262c62c80170aee33ebda2

I've also updated the default value of `textdomain` to be slug in 10b505291419fcc77831ea98a087b3cb07a9a3e0

Reply:
Perfect, thanks! Now we only have to update the examples to use `gutenpride` instead of `create-block`, at least in `wp-plugin.md`.

Reply:
okay, I think I've got all the examples updated to use `gutenpride` now :)